src/components/TablePreview/TablePreview.tsx

```diff
@@ -22,37 +23,38 @@ const TablePreview: FC<Props> = ({}) => {
         });
 
         navigator.clipboard.write([htmlData]);
-        console.log("Table copied!");
+        ToastQueue.positive("Table copied!", { timeout: 5000 });
```

Comment:
Cambiemos esto a que diga "Signature copied!" (el usuario no va a tener idea de que es un table behind the scenes)

```suggestion
        ToastQueue.positive("Signature copied!", { timeout: 5000 });
```